ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/RDXVisualizer.java

```diff
@@ -26,11 +32,31 @@ public abstract class RDXVisualizer implements RDXRenderableProvider
    private Consumer<Boolean> activenessChangeCallback;
    private boolean wasActive = false;
 
+   @Nullable
+   private ROS2Heartbeat heartbeat;
+
    public RDXVisualizer(String title)
    {
       this.title = ImGuiTools.uniqueLabel(title);
    }
 
+   public void createRequestHeartbeat(ROS2Node node, ROS2Topic<Empty> heartbeatTopic)
```

Comment:
What about `setDemandTopic` or something that uses the word "demand" so it's more clear it's part of the architecture we designed called the "demand graph".

Reply:
Well, what it's doing is literally creating a `ROS2Heartbeat` aka a "request heartbeat". I think my naming is more accurate to the function.